common/archiver/interface.go

```diff
@@ -78,8 +78,19 @@ type (
 
 	// HistoryArchiver is used to archive history and read archived history
 	HistoryArchiver interface {
-		Archive(context.Context, URI, *ArchiveHistoryRequest, ...ArchiveOption) error
+        // Archive is used to archive a Workflow's history. When the context expires the method should stop trying to archive.
+        // Implementors are free to archive however they want, including implementing retries of sub-operations. The URI defines
+        // the resource that histories should be archived into. The implementor gets to determine how to interpret the URI.
+        // The Archive method may or may not be automatically retried by the caller. ArchiveOptions are used
+        // to interact with these retries including giving the implementor the ability to cancel retries and record progress
+        // between retry attempts.
+        // This method will be invoked after a workflow passes its retention period.
+        Archive(context.Context, URI, *ArchiveHistoryRequest, ...ArchiveOption) error
+        // Get is used to access an archived history. When context expires this method should stop trying to fetch history.
+        // The URI identifies the resource from which history should be accessed and it is up to the implementor to interpret this URI.
+        // This method should emit api service errors - see the filestore as an example.
 		Get(context.Context, URI, *GetHistoryRequest) (*GetHistoryResponse, error)
```

Comment:
Can you try running `go fmt ./common` and see if that make any changes to alignment of comments here?

Reply:
I don't have formatting issues locally.
I ran `go fmt ./common` as suggested but nothing changed.

Reply:
OK I think I fixed this.